aries_vcx/src/plugins/anoncreds/credx_anoncreds.rs

```diff
@@ -56,7 +56,12 @@ impl IndyCredxAnonCreds {
 
         let ms_value = (&record).try_get("value")?;
         let ms_decimal = ms_value.try_as_str()?;
-        let ms_bn: BigNumber = BigNumber::from_dec(ms_decimal)?;
+        let ms_bn: BigNumber = BigNumber::from_dec(ms_decimal).map_err(|err| {
+            AriesVcxError::from_msg(
+                AriesVcxErrorKind::UrsaError,
+                format!("Failed to create BigNumber from value: {}, error: {}", ms_decimal, err),
```

Comment:
We're dealing with link secrets here, which are pretty much private keys, we probably want to be careful to not leak any of these values in error messages. in retrospect, i'd probably be cautious about leaking the stringified error too (incase that error string leaks the value). So maybe all of these error messages should not contain any variables, and just be constant strings. What do you think?

Reply:
good points, let me minimize content of these message